src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackService.java

```diff
@@ -55,7 +55,8 @@ public CallbackService(final SecurityUtils securityUtils,
         }
         final CallbackRequest callbackRequest = new CallbackRequest(caseDetails, caseDetailsBefore, caseEvent.getId(), ignoreWarning);
         final Optional<ResponseEntity<CallbackResponse>> responseEntity = sendRequest(url, CallbackResponse.class, callbackRequest);
-        return responseEntity.map(re -> Optional.of(re.getBody())).orElseThrow(() -> new CallbackException("Unsuccessful callback to " + url));
+        return responseEntity.map(re -> Optional.of(re.getBody())).orElseThrow(() ->
+            new CallbackException("Callback to service has been unsuccessful for event " + caseEvent.getId()));
```

Comment:
can we define this error message in a constant field so we can reuse it below? Minor, ignore if you want

Reply:
What about a LOG.WARN of `Unsuccessful callback to " + url`? it's ok to not expose it to the user, but the failing callback url is an important info we could log

Reply:
Furthermore, maybe better to ask a BA what would be the best message to send to the end user? Perhaps something like 'Could not contact external service this event caseEvent.getId() depends on'?

Reply:
added a log.info line with url, caseType and event information. Updated the message with event name instead of the event Id. I assume the service name Probate, Divorce etc should be somehow clear to the user.